packages/frontend/core/src/blocksuite/ai/components/chat-action-list.ts

```diff
@@ -3,19 +3,23 @@ import type {
   EditorHost,
   TextSelection,
 } from '@blocksuite/affine/block-std';
+import { GfxControllerIdentifier } from '@blocksuite/affine/block-std/gfx';
 import {
   type ImageSelection,
   NotificationProvider,
 } from '@blocksuite/affine/blocks';
+import { WithDisposable } from '@blocksuite/affine/global/lit';
+import { DisposableGroup } from '@blocksuite/global/slot';
```

Comment:
I think you should use `@blocksuite/affine/global/slot`. If it's not exported, we should export it instead of adding a package here.